lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2017,11 +2018,23 @@ instance LiftHandler ErrAdjustForFee where
                 , showT missing, " Lovelace."
                 ]
 
+instance LiftHandler ErrUTxOTooSmall where
+    handler = \case
+        ErrUTxOTooSmall minUtxoValue invalidUTxO ->
+            apiError err403 UtxoTooSmall $ mconcat
+                [ "I'm unable to construct the given transaction as some "
+                , "outputs or changes are too small! Each output and change is "
```

Comment:
small note: we shouldn't be talking about change here because there's nothing users can do about change. We are actually generating them and, having changes that are _too small_ here would mean that we messed up. Yet, for outputs, this error looks totally legit indeed.